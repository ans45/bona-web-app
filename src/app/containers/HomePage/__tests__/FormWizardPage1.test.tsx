import React from 'react';
import { render } from '@testing-library/react';
import { FormWizardPage1 } from '../FormWizardPage1';
import { Form } from 'react-final-form';

describe('<FormWizardPage1  />', () => {
  it('should match snapshot', () => {
    const { container } = render(
      <Form onSubmit={() => undefined}>{() => <FormWizardPage1 />}</Form>,
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
