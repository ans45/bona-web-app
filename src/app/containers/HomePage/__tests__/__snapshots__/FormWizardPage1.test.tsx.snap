// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormWizardPage1  /> should match snapshot 1`] = `
<svg
  class="MuiBox-root MuiBox-root-1"
>
  <circle
    cx="60"
    cy="60"
    fill="#F3F3F3"
    r="60"
  />
</svg>
`;
